services/scribe/backfill/contract_test.go

```diff
@@ -60,7 +62,10 @@ func (b BackfillSuite) TestFailedStore() {
 		StoreConcurrency:   1,
 		GetLogsRange:       1,
 	}
-	backfiller, err := backfill.NewContractBackfiller(chainConfig, contractConfig, mockDB, simulatedChainArr, b.metrics)
+	blockHeightMeter, err := metrics.NewHistogram(b.metrics.Meter(), fmt.Sprint("scribe_block_meter", chainConfig.ChainID), "block_histogram", "a block height meter", "blocks")
```

Comment:
yeah, the point of the nil provider is you don't have to pass these in in the tests